engine/src/main/java/org/camunda/bpm/engine/impl/telemetry/reporter/TelemetrySendingTask.java

```diff
@@ -67,14 +62,15 @@ protected void sendData() {
       // send data only if telemetry is enabled
       if (commandContext.getProcessEngineConfiguration().getManagementService().isTelemetryEnabled()) {
         try {
-          HttpPost request = new HttpPost(telemetryEndpoint);
           String telemetryData = JsonUtil.asString(data);
-          StringEntity requestBody = new StringEntity(telemetryData, StandardCharsets.UTF_8);
-          request.setHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON);
-          request.setEntity(requestBody);
-          HttpResponse response = httpClient.execute(request);
+          HttpResponse response = http.createRequest()
+              .url(telemetryEndpoint)
+              .post()
+              .contentType(MediaType.APPLICATION_JSON)
+              .payload(telemetryData)
+              .execute();
 
-          if (response == null || HttpStatus.SC_ACCEPTED != response.getStatusLine().getStatusCode()) {
+          if (response == null || response.getStatusCode() != 202) {
```

Comment:
Instead of hard coded value 202  you can use java.net.HttpURLConnection.HTTP_ACCEPTED.

Reply:
thanks for the hint